### PR TITLE
[Fix Post Blog] - Meshery 5000 star milestone

### LINKED ISSUE #5598: [Post Blog] - Meshery 5000 star milestone
Contributors and community members are encouraged to post on https://layer5.io/blog. A new blog post fitting the following description is requested to be written by a community member:

**Desired Title:** 
Meshery 5000 star milestone

**Topic:** 
Meshery 5000 star milestone - The journey so far

**Core points to convey in the post:** 

1. The Genesis of Meshery
2. Who is Meshery For?
3. Celebrating the Community
    - A Heartfelt Thank You
    - Acknowledging the CNCF
4. The Road Ahead: Inviting New Contributors
    - How to Get Started
5. Reflections on the Journey
6. The Journey Has Just Begun

---
<h3>Contributor Resources</h3>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).

### PR DESCRIPTION
**Pull request by sandramsc**
**Description**

This PR fixes #5598

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 439319697f3dcc40bcec1d792cadd7b8cbedf5ee at: https://6675c64e6a08c6ccfbeb4929--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 5e1590486cd8bb13d1648751b6cd777542c663b3 at: https://6675d2e3543aaedbe5ed753a--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit e86e5d5609a4a99db0eb3f8be664b216ba8dc1a7 at: https://6675dae5af4203009f07c95e--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit a470ad5f9e97da0a3f1149f810d5160c1e197097 at: https://6675e25402ce9306169beee9--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 76b05b1d50ec6a0852e4eccc3683935a84b0f66c at: https://6675e94c543aaeefa5ed73df--layer5.netlify.app

**Comment by sandramsc** [comment]
Tried a few times to get the heading "The Journey Has Just Begun" to align on the left, under the headings of the same size above it by using br and p tag to as well as removing the mentioned tags, but it remains aligned on the left. Feedback on how to adjust/fix this is welcome.

imageedit_4_7999326101

**Comment by vishalvivekm** [comment]
Thank you for your contribution.
Let's discuss this on Websites's call. Add this as an agenda item into the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit), if you would :)

**Comment by leecalcote** [comment]
Argh! I should have reviewed this years ago. 🫤 Thank you for this, @sandramsc 👍

**Comment by l5io** [comment]
🚀 Preview for commit 11fc7575a45a8a094d956371150d360e11c95323 at: https://66a00ef4ad6e3933fe04125f--layer5.netlify.app

File: src/collections/blog/2024/06-21-meshery-5000-star-milestone/index.mdx
Comment (hargunkaur286) [inline src/collections/blog/2024/06-21-meshery-5000-star-milestone/index.mdx:1]: 1. you can make the

**Review by hargunkaur286 — CHANGES_REQUESTED** [review]
Overall, the content looks good to me!

**Comment by sudhanshutech** [comment]
@sandramsc does hargun's feedback looks good?

**Comment by leecalcote** [comment]
@hargunkaur286 please apply these formatting changes on @sandramsc's behalf. 👍

**Pull request merged**